features/steps/misc.py

```diff
@@ -84,3 +85,11 @@ def then_folder_does_not_exist(context, folder):
         user_spec="with sudo",
     )
     assert_that(context.process.stdout.strip(), equal_to("empty"))
+
+
+@when("I regexify `{var_name}` stored var")
```

Comment:
My new favorite verb: "regexify" :D

Reply:
haha I still don't know if that is real word or not :man_shrugging: